src/power_meter.h

```diff
@@ -0,0 +1,74 @@
+#ifndef __POWER_METER_H__
+#define __POWER_METER_H__
+
+#include <ModbusMaster.h>
+
+// Implemented for use with PZEM-014
+
+class PowerMeter {
+ public:
+  PowerMeter(uint8_t device_address, int pin_de, int pin_re);
+  ~PowerMeter();
+
+  // TODOLater - Determine if calibration function is required
```

Comment:
The online code doesn't include any calibration code, so it may or may not be needed. Could probably be written quickly if it is needed